recipes/g2clib/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+    name: g2clib
+    version: 1.5.0
+
+source:
+    fn: g2clib-1.5.0-patch.tar.gz
+    url: http://www.ncl.ucar.edu/Download/files/g2clib-1.5.0-patch.tar.gz
+    md5: f857f49a66918316996eb52399e04120
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - libpng
+        - zlib
+        - jpeg
+        - jasper
+        - gcc
+
+test:
+    commands:
+        - exit $(test -f ${PREFIX}/lib/libgrib2c.a)  # [not win]
```

Comment:
We can probably do much better with `conda inspect`, but for now I went lazy and added a simple test to assert the lib exists.